esp/src/src-react/components/Metrics.tsx

```diff
@@ -625,7 +629,12 @@ export const Metrics: React.FunctionComponent<MetricsProps> = ({
                 <DockPanel layout={options?.layout} onDockPanelCreate={setDockpanel}>
                     <DockPanelItem key="scopesTable" title={nlsHPCC.Metrics}>
                         <HolyGrail
-                            header={<SearchBox value={scopeFilter} onChange={onChangeScopeFilter} iconProps={filterIcon} placeholder={nlsHPCC.Filter} />}
+                            header={<Stack horizontal>
+                                <Stack.Item grow>
+                                    <SearchBox value={scopeFilter} onChange={onChangeScopeFilter} iconProps={filterIcon} placeholder={nlsHPCC.Filter} />
+                                </Stack.Item>
+                                <ToggleButton appearance="subtle" icon={<TextCaseTitleRegular />} title={nlsHPCC.MatchCase} checked={matchCase} onClick={() => { setMatchCase(!matchCase); }} />
```

Comment:
Only thing I noticed when I checked this branch out is the state of this button is pretty subtle. I thought maybe `icon={matchCase ? <TextCaseTitleRegular /> : <TextCaseLowercaseRegular />}`?